pkg/oc/bootstrap/docker/up.go

```diff
@@ -726,6 +730,18 @@ func (c *CommonStartConfig) CheckDockerVersion(out io.Writer) error {
 	return nil
 }
 
+// CheckPortForwardingPrerequisites checks that socat is installed when port forwarding is enabled
+// Socat needs to be installed manually on MacOS
+func (c *CommonStartConfig) CheckPortForwardingPrerequisites(out io.Writer) error {
+	err := localcmd.New("socat").Args("-V").Run()
```

Comment:
why not use https://golang.org/pkg/os/exec/#LookPath ?

Reply:
I wanted to cover the case when users installed that using brew and have it in the path but it was missing some additional packages. It happens randomly on some machines.
```
socat
dyld: Library not loaded: /usr/local/opt/readline/lib/libreadline.7.dylib
  Referenced from: /usr/local/bin/socat
  Reason: image not found
[1]    13015 abort      socat 
```